Alignment/CTPPS/interface/StraightTrackAlignment.h

```diff
@@ -55,7 +57,7 @@ class StraightTrackAlignment
 
     virtual void Begin(const edm::EventSetup&);
 
-    virtual void ProcessEvent(const edm::DetSetVector<TotemRPRecHit> &hitsStrip, const edm::DetSetVector<CTPPSDiamondRecHit> &hitsDiamond,
+    virtual void ProcessEvent(const edm::DetSetVector<TotemRPUVPattern> &UVPattern, const edm::DetSetVector<CTPPSDiamondRecHit> &hitsDiamond,
```

Comment:
Please use variable names starting with lower case. In this case please use a name that indicates its relation to strips, e.g. `uvPatternsStrip`.